docs/inferences/model-runner.md

```diff
@@ -0,0 +1,40 @@
+Running inferences with the standard `landingai.predict.Predictor` will send your image to LandingLens cloud, which is ideal if you don't want to worry about backend scalability, hardware provisioning, availability, etc. But this also adds some networking overhead that might limit how many inferences per second you can run.
+
+If you need to run several inferences per second, and you have your own cloud service or local machine, you might want to run inference using your own resources. For that, we provide **[Model Runner](https://support.landing.ai/docs/docker-deploy)**, a Docker image with your LandingLens trained model embeded that you can run anywhere.
+
+
+???+ note
+
+    You can get more details on how to setup and run Model Runner container locally or in your own cloud service in our [Support Center](https://support.landing.ai/docs/docker-deploy).
```

Comment:
Rephrase the first part to "You can get more details on how to set up and run the Docker deployment container locally..." (In this context, "set up" is 2 words".

Reply:
Done! Thanks!